### PR TITLE
ReadMe Improvements

### PR DESCRIPTION
**Pull request by tomspilman**
This is a work in progress.

Some more improvements to the read me for developers and users alike.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
+| Package NuGet | [![Build Status](http://teamcity.monogame.net/app/rest/builds/buildType:MonoGame_PackageNuGet/statusIcon)](http://teamcity.monogame.net/viewType.html?buildTypeId=MonoGame_PackageNuGet&guest=1) |
+| Package Mac and Linux | [![Build Status](http://teamcity.monogame.net/app/rest/builds/buildType:MonoGame_PackageMacAndLinux/statusIcon)](http://teamcity.monogame.net/viewType.html?buildTypeId=MonoGame_PackageMacAndLinux&guest=1) |
+| Package Windows | [![Build Status](http://teamcity.monogame.net/app/rest/builds/buildType:MonoGame_PackagingWindows/statusIcon)](http://teamcity.monogame.net/viewType.html?buildTypeId=MonoGame_PackagingWindows&guest=1) |
 
-Desktop:
+
+## Supported Platforms
+We support a growing list of platforms across the desktop, mobile, and console space.  If there is a platform we don't support, please [make a request](https://github.com/mono/MonoGame/issues) or [come help us](CONTRIBUTING.md) add it.
+
+- Desktops
```
Comment (cra0zy) [inline README.md]: I think "Desktop" makes more sense than "Desktops".

**Comment by tomspilman** [comment]
Going to flatten these commits a bit... hold on.

**Comment by tomspilman** [comment]
Ok... good to go if it looks right.

**Comment by cra0zy** [comment]
The submodule command is not needed for getting the source code because Protobuild autodownloads submodules if they are not present.

**Comment by cra0zy** [comment]
I see few other things, I'll do one small update of stuff...

**Comment by tomspilman** [comment]
> because Protobuild autodownloads submodules if they are not present.

I am aware of that...  but I don't like depending on it.  That is why I added those details.

**Pull request merged**